web_src/js/features/dropzone.ts

```diff
@@ -128,10 +128,12 @@ export async function initDropzone(dropzoneEl) {
       fileUuidDict = {};
       for (const attachment of respData) {
         const file = {name: attachment.name, uuid: attachment.uuid, size: attachment.size};
-        const imgSrc = `${attachmentBaseLinkUrl}/${file.uuid}`;
         dzInst.emit('addedfile', file);
-        dzInst.emit('thumbnail', file, imgSrc);
         dzInst.emit('complete', file);
+        if (isImageFile(attachment.name)) {
```

Comment:
Super small nit: the code below and above this line seem to use `file` from here on out, so this could be `file.name` for consistency

Reply:
> const file = {name: attachment.name, uuid: attachment.uuid, size: attachment.size};

They are equal.

https://github.com/go-gitea/gitea/pull/32319/commits/cd9febc522c93f29c208aade90f86b091cded30d